framework/base/Module.php

```
@@ -689,7 +689,9 @@ public function runAction($route, $params = [])
     protected function runMappedAction(string $id, array $params): mixed
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)